README.md

```diff
@@ -40,6 +38,23 @@ const server = new Pretender(function() {
 $.get('/photos/12', {success() => { ... }})
 ```
 
+## Usage examples
+
+You can load Pretender directly in the browser.
+
+```javascript
+<script src="pretender.js"></script>
+```
+
+Or as a module.
+
+```javascript
+import Pretender from 'pretender';
+const server = new Pretender(function() {});
+```
+
+Full example: [use-pretender-as-a-module](https://github.com/givanse/use-pretender-as-a-module)
```

Comment:
For the repo linked here, I added as collaborators the people that have merge access to Pretender.

Or if you prefer me to delete the mention of 'use-pretender-as-a-module', that is fine, I will.